Sources/StreamChat/ChatClient+Environment.swift

```diff
@@ -165,12 +165,14 @@ extension ChatClient {
         var offlineRequestsRepositoryBuilder: (
             _ messageRepository: MessageRepository,
             _ database: DatabaseContainer,
-            _ apiClient: APIClient
+            _ apiClient: APIClient,
+            _ retryHoursThreshold: CGFloat
         ) -> OfflineRequestsRepository = {
             OfflineRequestsRepository(
                 messageRepository: $0,
                 database: $1,
-                apiClient: $2
+                apiClient: $2,
+                retryHoursThreshold: $3
```

Comment:
Why not simply `retryThreshold` and allow the pass in seconds (like most Foundation APIs)?

Reply:
thought about it, but this one doesn't require such precision and hours should be easier to set for customers. I'm open to changing it if everyone thinks it's better the other way.

Reply:
IMO I think in this one, it makes sense to be in hours because it should be easier to use for customers. But I think it should be an `Int` or, at most `Float` and not `CGFloat`, since the latter is for CoreGraphics, and this is a config for LLC, not UI Kit. `Int` should be enough tho, since it does not make sense to select 3 and a half hours for example...

Reply:
makes sense, changed this

Reply:
I would argue that the precision isn't needed as we don't know all the requirements people may have. Making it in seconds it's flexible while maintaining simplicity. The type would also need to be TimeInterval. Not a blocker ofc just stating my side of thinking.